polling_stations/settings/constants/importers.py

```diff
@@ -1,20 +1,5 @@
-"""
-Amazon S3 config:
-By default, we will look for a section
-
-[wheredoivote]
-aws_access_key_id = xxxx
-aws_secret_access_key = xxxx
-
-in a /etc/boto.cfg or ~/.boto file, etc
-See: http://boto.cloudhackers.com/en/latest/boto_config_tut.html
-
-We can change the section name using the BOTO_SECTION setting
-"""
 import os
 
-BOTO_SECTION = "wheredoivote"
-
```

Comment:
I've updated this so that it will work with `aws sso login` (or roles when deployed).
We now have to configure this with env vars. So for example, if I want to work on import scripts locally (without `PRIVATE_DATA_PATH` set) I would do:

```
aws sso login --profile wdiv-prod
AWS_PROFILE=wdiv-prod ./manage.py import_council_x
```

(assuming my profile is called wdiv-prod in my `.aws/config` file)

or if I want to work on the uploader locally, uploading to the dev s3 bucket I could do

```
aws sso login --profile wdiv-dev
AWS_PROFILE=wdiv-dev ./manage.py runserver
```

Reply:
Yes. Having to set `S3_DATA_BUCKET` isn't new but you're right none of this is documented. It is one of those awkward "this is a public codebase but also we need to document things about how we internally use it" things. I will write something about
- this
- PRIVATE_DATA_PATH
- S3_DATA_BUCKET

and chuck it in the wiki